cmd/tether/main_linux.go

```diff
@@ -38,6 +38,10 @@ func main() {
 		halt()
 	}()
 
+	logFile, _ := os.OpenFile("/dev/ttyS1", os.O_WRONLY|os.O_SYNC, 0644)
+	syscall.Dup2(int(logFile.Fd()), 2)
```

Comment:
Actually @caglar10ur I only made this change to vic-init (which is Photon) and tether-linux, which also runs on Photon -- are you sure that there needs to be a concern here regarding architectures?